contents/newsletter/talk-to-users.md

```diff
@@ -0,0 +1,106 @@
+
+A fix for this is spending more time learning what users want. Nothing beats doing this yourself directly. By doing this, you gain the knowledge to that closes the gap between what users value and what you are building.
+
+![Don't build the wrong features](https://res.cloudinary.com/dmukukwp6/image/upload/v1713470851/posthog.com/contents/images/newsletter/talk-to-users/features.jpg)
+
+## Finding users to talk with
+
+Just because we convinced you that talking to users isn't a waste of time does not mean we want to waste your time doing so. We realize engineers are busy, but that isn't an excuse to not talk to users. As Farley writes:
+
+> Modern teams fight with schedule pressure, quality, and maintainability of their designs. They often struggle to identify the ideas that really land with users, and they fail to allow themselves the time to learn about the problem domain, the technology, and the opportunities to get something great into production.
+
+The lucky for you, you don't need a lot of users. This doesn't require a statistically significant sample. A few user interviews can be all you need to clarify the next increment of progress.
+
+Second, you don't need to go send cold emails to find users to speak with. Once you have [product-market fit](https://posthog.com/founders/product-market-fit-game), your current users work perfectly. From our experience, these are the best ways to find them:
+
+- Look at analytics to find power users of related features.
+- Ask your sales or customer success teams for potentially interested customers.
+- Check support for who is asking or requesting features.
+- Your users could also be inside your company. For example, our infrastructure team talks to our product teams to understand their upcoming requirements.
+
+To speed up the time it takes to find users, use a tool like PostHog's [user interview survey](https://posthog.com/tutorials/feedback-interviews-site-apps) or provide an incentive, like a merch code or gift card. 
+
+![Feedback for merch code](https://res.cloudinary.com/dmukukwp6/image/upload/v1713470870/posthog.com/contents/images/newsletter/talk-to-users/merch.png)
+
+## What NOT to talk about
+
+Ironically enough, information from talking to users comes from **NOT TALKING**. Beyond prioritizing listening, there are some specific areas you should avoid. They include:
+
+1. **Yourself or your product.** If they are users, they probably know what your product is and what it does.
+
+2. **Your solution.** Resist the urge to explain your features and your thinking. You shouldn't use the time to convince them why your solution is the right one.
+
+3. **Their solutions.** Users likely recommendations and feature requests. These often don't solve their underlying problems, so reframe these to dig deeper into the underlying causes.
```

Comment:
is there a missing word here?

 _Users likely recommendations and feature requests_